rust-runtime/aws-smithy-protocol-test/src/lib.rs

```diff
@@ -474,6 +475,17 @@ actual body in base64 (useful to update the protocol test):
     }
 }
 
+pub fn decode_body_data(body: &[u8], media_type: MediaType) -> Cow<'_, [u8]> {
```

Comment:
Can you change the server protocol test generator to also rely on this function? I basically code-generated the same thing when generating the tests:

https://github.com/smithy-lang/smithy-rs/blob/c63e79245430c98b2725627566b80a0fbf8e9619/codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/protocol/ServerProtocolTestGenerator.kt#L458-L462

Reply:
Addressed in 6ef41eb